PRS4001-5000/PRS4133Flad.xml

```diff
@@ -50,28 +50,32 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <listPerson>
             <person sex="1" sameAs="Q93086916">
                <persName>
-                  <forename>Johann Martin</forename>
+                  <forename>Johann</forename>
+                     <forename>Martin</forename>
                   <surname>Flad</surname>
                </persName>
                <occupation type="ecclesiastic">missionary</occupation>
                <faith type="Protestantism"/>
                <birth when="1831"/>
-               <floruit notBefore="1855" notAfter="1894">
-                  He was one of the most prominent Protestant
-                  missionaries to <placeName ref="LOC3010Ethiop">Ethiopia</placeName> in the nineteenth century, making numerous trips there
-                  between 1855 and 1894 and being among the captives at <placeName
-                     ref="LOC4547Maqdal"/> freed by the <persName
-                     ref="PRS7484Napier"/> expedition. He was active in Bible work,
-                  promotion of Christianity among <persName ref="ETH1274Betaes"/>  and, occassionally, purchase of manuscripts. </floruit>
+               <floruit notBefore="1855" notAfter="1894"> He was one of the most prominent
+                  Protestant missionaries to <placeName ref="LOC3010Ethiop">Ethiopia</placeName> in
+                  the nineteenth century, making numerous trips there between 1855 and 1894 and
+                  being among the captives at <placeName ref="LOC4547Maqdal"/> freed by the
+                     <persName ref="PRS7484Napier"/> expedition. He was active in Bible work,
+                  promotion of Christianity among <persName ref="ETH1274Betaes"/> and,
+                  occassionally, purchase of manuscripts. </floruit>
                <death when="1915"/>
 
             </person>
             <listRelation>
-               <relation name="betmas:husbandOf" active="PRS4132Flad" passive="PRS4134Flad"></relation>
-               <relation name="snap:ProfessionalRelationship" active="PRS4132Flad" passive="PRS6064Krapf"></relation>
-               <relation name="snap:ProfessionalRelationship" active="PRS4132Flad" passive="PRS4722Gobat"></relation>
-               
-               <relation name="snap:FriendshipFor" active="PRS4132Flad" passive="PRS7081Mikael"><desc> Mikāʾel ʾAragāwi was Flad´s first adopted son.</desc>
+               <relation name="betmas:husbandOf" active="PRS4132Flad" passive="PRS4134Flad"/>
+               <relation name="snap:ProfessionalRelationship" active="PRS4132Flad"
+                  passive="PRS6064Krapf"/>
+               <relation name="snap:ProfessionalRelationship" active="PRS4132Flad"
+                  passive="PRS4722Gobat"/>
+
+               <relation name="snap:FriendshipFor" active="PRS4132Flad" passive="PRS7081Mikael">
```

Comment:
I wonder, if snap:FatherOf could be the better choice.

Reply:
I did not understand the relationship correctly. I think snap:FatherOf only applies to familial paternity, i.e. someone is the biological father or someone has adopted a child and treated it as if it were their own biological child.
For a spiritual connection, I suggest dc:relation with a short note in desc to describe this relationship precisely.

Reply:
> I did not understand the relationship correctly. I think snap:FatherOf only applies to familial paternity, i.e. someone is the biological father or someone has adopted a child and treated it as if it were their own biological child. For a spiritual connection, I suggest dc:relation with a short note in desc to describe this relationship precisely.

thank you! I will follow you suggestion then! I yill not provide any note, because I am afraid of being wrong :-)

Reply:
dc:relation is a very general and not specific relation. It needs a description (desc) to explain it.